### DISCUSSION TITLE
Make `*computed*` functions consistent

### DISCUSSION THREAD

**Discussion by bundfussr** [pharmaverse/admiral#2179] [Ideas]
Currently the behavior of `derive_param_computed()` differs from all other functions: if `dataset` and `dataset_add` are specified, both datasets are used as input.

To make it consistent `dataset_add` should be mandatory and only this dataset should be used to derive the new records.

For the use case that more than one source dataset is required like for some questionnaires parameters `derive_param_computed_msrc()` should be implemented.
```
derive_param_computed_msrc( 
  dataset = NULL, 
  by_vars, 
  parameters, # list of parameter() objects 
  source_datasets, 
  set_values_to, 
  constant_by_vars = NULL, 
  constant_parameters = NULL, 
  keep_nas = FALSE 
) 

parameter( 
  dataset_name, 
  id, 
  condition = NULL 
) 
```

**Comment by bundfussr** [comment]
@manciniedoardo , why was this closed?

Wouldn't this be a good change for admiral 2.0?

**Reply by manciniedoardo** [reply]
sorry, i thought this fell under the review we did prior to 1.1 - will reopen
